config/jobs/periodic/docker-in-docker/periodic-dind-build.yaml

```diff
@@ -0,0 +1,213 @@
+periodics:
+  - name: docker-build
+    labels:
+      preset-secrets: "true"
+    cluster: k8s-ppc64le-cluster
+    decorate: true
+    interval: 24h
+    spec:
+      containers:
+      - image: quay.io/florencepascual/docker_ce_build
+        command:
+        - /bin/bash
+        args:
+        - -c
+        - |
+          # path to the github repository
+          DIR_GITHUB="docker-ce-build"
+          PATH_GITHUB="https://github.com/florencepascual/${DIR_GITHUB}.git"
+          
+          # path to the image for building and testing
+          PATH_IMAGE_BUILD="quay.io/florencepascual/docker_ce_build"
+
+          # path to the scripts 
+          PATH_SCRIPTS="/workspace/${DIR_GITHUB}"
+          DATE=`date +%d%m%y-%H%S`
+          PATH_LOG_PROWJOB="/workspace/prow-job-${DATE}.log"
+
+          export DIR_GITHUB
+          export PATH_SCRIPTS
+          export PATH_LOG_PROWJOB
+
+          # Create the log file
+          echo "Create the prow job log file"
+          touch ${PATH_LOG_PROWJOB}
+          if ! test -f ${PATH_LOG_PROWJOB}
+          then
+            echo "The log file ${PATH_LOG_PROWJOB} was not created."
+            exit 1
+          fi
+
+          # clone the directory where the scripts are
+          echo "* Git clone and get the dockerd-entrypoint script*" 2>&1 | tee -a ${PATH_LOG_PROWJOB}
```

Comment:
Use extra_refs for cloning (You may remove `git clone` and related check condition with this)

Reply:
Ok, I will change that